end-to-end-test/shared/wdio/webdriver-manager.conf.js

```diff
@@ -70,6 +70,7 @@ var config = {
                 args: [
                     '--disable-composited-antialiasing',
                     '--allow-insecure-localhost',
+                    '--no-sandbox',
```

Comment:
what's this?

Reply:
this is the attempt at fixing the circleCI issue